dev/MRTCore/mrt/Core/src/MRM.vcxproj

```diff
@@ -104,7 +104,7 @@
            supported according to the CRT maintainer. Dynamic linking against the CRT makes the binaries a bit smaller
            than they would otherwise be if the CRT, runtime, and STL were all statically linked in. -->
       <IgnoreSpecificDefaultLibraries>%(IgnoreSpecificDefaultLibraries);libucrtd.lib</IgnoreSpecificDefaultLibraries>
-      <AdditionalOptions>%(AdditionalOptions) /defaultlib:ucrtd.lib</AdditionalOptions>
+      <AdditionalOptions>%(AdditionalOptions) /defaultlib:ucrtd.lib /debug /debugtype:cv,fixup /opt:ref /opt:icf /incremental:no</AdditionalOptions>
```

Comment:
Is APIScan only done on debug build? what if release build does something different (although it's not recommended)?

Reply:
the /debug switch here doesn't refer to the build configuration
it refers to whether a pdb should be generated
for the question of whether we want the binaries to be ready for apiscan for one or both build configurations, jon thought perhaps we can do just release (so that debug builds retain incrementality) and asked me to check with ben
iteration 2 does that: it preps the binaries for apiscan only for the release configuration
i'll check with ben if that's ok